source/change-streams/tests/README.rst

```diff
@@ -158,4 +158,41 @@ The following tests have not yet been automated, but MUST still be tested
 #. The ``killCursors`` command sent during the "Resume Process" must not be allowed to throw an exception.
 #. ``$changeStream`` stage for ``ChangeStream`` against a server ``>=4.0`` that has not received any results yet MUST include a ``startAtOperationTime`` option when resuming a changestream.
 #. ``ChangeStream`` will resume after a ``killCursors`` command is issued for its child cursor.
-
+#. ``ChangeStream`` against a server ``>=4.2`` must track the latest ``postBatchResumeToken`` included in an ``aggregate`` or ``getMore`` response.
+#. - For a ``ChangeStream`` under these conditions:
+      - Running against a server ``>=4.2``.
+      - The batch is empty or has been iterated to the last document.
+   - Expected result: 
+       - The ``getResumeToken`` must return the ``postBatchResumeToken`` from the current command response.
+#. - For a ``ChangeStream`` under these conditions:
+      - Running against a server ``<4.2``.
+      - The batch is empty or has been iterated to the last document.
+   - Expected result: 
+       - The ``getResumeToken`` must return the ``startAfter`` or ``resumeAfter`` from the initial aggregate. 
+       - If ``startAfter`` and ``resumeAfter`` are absent, ``getResumeToken`` result must be empty.
+#. - For a ``ChangeStream`` under these conditions:
+      - The batch is not empty.
+      - The batch has been iterated up to but not including the last element.
+   - Expected result:
+      - The ``getResumeToken`` must return the last cached document resume token.
+#. - For a ``ChangeStream`` under these conditions:
+      - The batch is not empty.
+      - The batch hasn’t been iterated at all.
+      - Executing an ``aggregate`` command.
+   - Expected result:
+      - The ``getResumeToken`` must return the ``startAfter`` or ``resumeAfter`` from the initial ``aggregate`` request.
```

Comment:
Same suggestion as my previous suggestion about splitting this into three bullets and capturing the precedence of `startAfter`, then `resumeAfter`, and finally an empty return value.

Reply:
Done